history.txt

```diff
@@ -5,6 +5,26 @@
 OMERO version history
 =====================
 
+5.4.6-rc1 (April 2018)
+----------------------
+
+This is a bug-fix release which also introduces a significant
```

Comment:
So, in fact, it is not a bug-fix release in the first place. Start this paragraph with the new feature, then quickly mention ... and some bug-fixes.